src/plonk/vanishing/verifier.rs

```diff
@@ -64,45 +70,45 @@ impl<F: WithSmallOrderMulGroup<3>, CS: PolynomialCommitmentScheme<F>> Committed<
 impl<F: WithSmallOrderMulGroup<3>, CS: PolynomialCommitmentScheme<F>> Constructed<F, CS> {
     pub(in crate::plonk) fn evaluate_after_x<T: Transcript>(
         self,
-        vk: &VerifyingKey<F, CS>,
         transcript: &mut T,
-    ) -> Result<Evaluated<F, CS>, Error>
+    ) -> Result<PartiallyEvaluated<F, CS>, Error>
     where
         F: Hashable<T::Hash>,
     {
-        let h_evals = read_n(transcript, vk.domain.get_quotient_poly_degree())?;
         let random_eval = transcript.read()?;
```

Comment:
BTW IIRC there is no real reason to read the random evals here and to have them be part of that struct, could be cleaner to do it right before the x_3 challenge and call it `randomness_before_x3` or something :) (not very important though)